public/app/features/canvas/runtime/element.tsx

```diff
@@ -29,6 +29,11 @@ export class ElementState implements LayerElement {
   // Determine whether or not element is in motion or not (via moveable)
   isMoving = false;
 
+  // Determine whether or not to use temp stored constraint for visualization purposes
+  // (switch to top / left constraint to simplify some functionality)
+  shouldUseTempConstraint = false;
+  tempConstraint: Constraint = {};
```

Comment:
Maybe rather than boolean and constraint, we could just use:
```suggestion
  tempConstraint: Constraint | undefined;
```
if set, we use it